src/SFML/Audio/SoundFileReaderWav.cpp

```diff
@@ -65,6 +65,17 @@ namespace
         return true;
     }
 
+    bool decode(sf::InputStream& stream, sf::Int32& value)
+    {
+        unsigned char bytes[sizeof(value)];
+        if (stream.read(bytes, sizeof(bytes)) != sizeof(bytes))
```

Comment:
Are the whole `sizeof`s not a bit confusing?

`sizeof(sf::Int32)` is 4 by definition, and in line 74, you assume 4 bytes anyway. I think the code might be more readable with a hardcoded `4`. Now it feels like the actual number of bytes could change, which is not true -- if it were, then we should seriously reconsider the definition of `sf::Int32` type.